### PR TITLE
Update docs to reflect changes in Docker images labels format

### PR DESCRIPTION
**Pull request by mfojtik**
@bparees @soltysh PTAL

### CODE REVIEW

**Comment by mfojtik** [comment]
@adellape

File: creating_images/metadata.adoc [hunk 1]
```diff
@@ -30,18 +30,21 @@ information on the `LABEL` instruction.
 
 The label names should typically be namespaced. The namespace should be set
 accordingly to reflect the project that is going to pick up the labels and use
-them. For OpenShift the namespace should be set to *_openshift.io/_* and for
-Kubernetes the namespace is *_k8s.io/_*. For simple labels, like `displayName`
+them. For OpenShift the namespace should be set to *_io.openshift/_* and for
```
Comment (soltysh) [inline creating_images/metadata.adoc]: I thought we agreed to get rid of the slash `/` in the labels.

Comment (soltysh) [inline creating_images/metadata.adoc]: And you actually removed those in the following lines.

**Comment by mfojtik** [comment]
@soltysh thanks! fixed

**Comment by soltysh** [comment]
LGTM

File: creating_images/metadata.adoc [hunk 2]
```diff
@@ -30,18 +30,21 @@ information on the `LABEL` instruction.
 
 The label names should typically be namespaced. The namespace should be set
 accordingly to reflect the project that is going to pick up the labels and use
-them. For OpenShift the namespace should be set to *_openshift.io/_* and for
-Kubernetes the namespace is *_k8s.io/_*. For simple labels, like `displayName`
+them. For OpenShift the namespace should be set to *_io.openshift_* and for
+Kubernetes the namespace is *_io.k8s_*. For simple labels, like `display-name`
```
Comment (bparees) [inline creating_images/metadata.adoc]: given an example of namespacing.

**Comment by adellape** [comment]
@mfojtik It looks like this change is only applicable to Origin so far, yes? If so, we'll need to conditionalize these between Enterprise and Origin with ifdefs, until Enterprise is in sync.

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/contributing.adoc#conditional-text-between-products

**Comment by mfojtik** [comment]
@adellape it is applicable to both... we are keeping the released images (rhel7 based) backward compatible, but they will also receive the new labels set.

**Pull request merged**